resources/quality/blocks/Nozzle/blocks_global_0.20_standard.inst.cfg

```diff
@@ -0,0 +1,18 @@
+[general]
+version = 4
+name = Standard Quality
+definition = blocks_base
+
+[metadata]
+setting_version = 19
+type = quality
+quality_type = normal 0.2
+weight = -3
+global_quality = True
+variant = 0.2mm Nozzle
+
+[values]
+layer_height = 0.1
+layer_height_0 = 0.15
+material_diameter = 1.75

```

Comment:
This should not be set in a quality profile. It's already being set by the extruder. THese can (and should) be removed from all of them.